SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -61,12 +61,23 @@ struct FConnectionConfig
 struct FReceptionistConfig : public FConnectionConfig
 {
 	FReceptionistConfig()
-		: ReceptionistHost(SpatialConstants::LOCAL_HOST)
-		, ReceptionistPort(SpatialConstants::DEFAULT_PORT)
+		: ReceptionistPort(SpatialConstants::DEFAULT_PORT)
 	{
 		const TCHAR* CommandLine = FCommandLine::Get();
 
-		FParse::Value(CommandLine, TEXT("receptionistHost"), ReceptionistHost);
+		// Parse the commandline for receptionistHost, if it exists then use this as the host IP.
+		if (!FParse::Value(CommandLine, TEXT("receptionistHost"), ReceptionistHost))
+		{
+			// If receptionistHost is not specified then parse for a traditional IP.
+			if (!FParse::Token(CommandLine, ReceptionistHost, 0) || **ReceptionistHost == '-' || **ReceptionistHost == '/')
```

Comment:
Use to array operator to get the first character.

Reply:
It's called the bracket operator MIKE PLEASE